### ISSUE TITLE
[en] "flashmob" word isn't known

### ISSUE THREAD

**Issue by donnerpeter** [languagetool-org/languagetool#7328]

**Comment by tiff** [comment]
All dictionaries except for Wiktionary only know "flash mob"

**Comment by donnerpeter** [comment]
True. Still, Google gives me 5M results for `"flashmob"` vs 7.5M for `"flash mob"`: not a huge difference. CNN used it, at least in 2009: http://edition.cnn.com/2009/WORLD/europe/02/09/uk.station.flashmob/index.html

But it's for you to decide, of course :)

**Comment by Tex2002ans** [comment]
> All dictionaries except for Wiktionary only know "flash mob"

Cambridge has "flashmob":

- [Cambridge: "flashmob"](https://dictionary.cambridge.org/us/dictionary/english/flashmob)

Collins has "flashmobbing":

- [Collins: "flash mob"](https://www.collinsdictionary.com/us/dictionary/english/flash-mob)

Dictionary.com has "flash mob" OR "flashmob":

- [Dictionary.com: "flash mob"](https://www.dictionary.com/browse/flash-mob)

If you look at:

- [Google n-grams: "flash mob" vs. "flashmob"](https://books.google.com/ngrams/graph?content=flashmob%2Cflash+mob&year_start=1800&year_end=2019&corpus=26&smoothing=3)

you see both appeared ~2000, and "flashmob" is used ~1/10th as often as "flash mob".

- - -

**Note:** OED seems to have "flash mob" OR "flashmob" as well.

(Although I can't verify that one since I don't have an OED account, but Google pops up a definition box + cites OED.)

There's also the odd "flash mobber" / "flash mobbers":

- [Oxford Learner's Dictionary: "flash mobber"](https://www.oxfordlearnersdictionaries.com/definition/english/flash-mobber)

but that one looks to be used with a space:

- flash mobber = ~30 million hits
- flashmobber = ~6.3k hits

- - -

So, I suspect these 4 words *could* be added:

- flashmob
- flashmobs
- flashmobbed
- flashmobbing
